packages/adapter-node/src/server.js

```diff
@@ -29,8 +29,12 @@ export function createServer({ render }) {
 
 	const assets_handler = fs.existsSync(paths.assets)
 		? sirv(paths.assets, {
-				maxAge: 31536000,
-				immutable: true,
+				setHeaders: (res, pathname, stats) => {
+					// eslint-disable-next-line no-undef
+					if (pathname.startsWith(`/${esbuild_app_dir}/`)) {
```

Comment:
This will require that the user not prefix or suffix `config.kit.appDir` with `/`. I think it'd be good to add either some sanitation or validation. E.g. we could automatically strip that prefix / suffix character. Or we could assert its value as we're doing for the base path in https://github.com/sveltejs/kit/commit/dc56d3cef4ec0aca3d074757c4a15d1f640c5c56

Reply:
@benmccann added check for leading slash + app_dir as root

Reply:
and app_dir can not be empty string

Reply:
But maybe these checks are unnecessary? SvelteKit doesn't work with `app_dir: "/"` or `app_dir: "/my_app_dir"` anyway, in `svelte.config.js`.

Reply:
That should probably be fixed in another PR.

Reply:
It still looks like it will not work if someone specifies `config.kit.appDir` with a trailing slash

Reply:
aaa stupid me...

Reply:
@benmccann fixed